appveyor.yml

```diff
@@ -27,13 +24,19 @@ platform:
   
 environment:
   matrix:
+   - APPVEYOR_BUILD_WORKER_IMAGE: Visual Studio 2015
+     SQLITE_ORM_CXX_STANDARD: ""
+
    # using c++14
-   - SQLITE_ORM_CXX_STANDARD: "-DSQLITE_ORM_ENABLE_CXX_14=ON"
+   - APPVEYOR_BUILD_WORKER_IMAGE: Visual Studio 2022

```

Comment:
There's still something wrong with the environment matrix: appveyor shouldn't build using Visual Studio 2022 with no `SQLITE_ORM_CXX_STANDARD` at all. [See appvyeor build](https://ci.appveyor.com/project/fnc12/sqlite-orm/builds/43393952).

Reply:
stranger things. Let's try to figure it out

Reply:
I am looking at appveyor's [build environment doco](https://www.appveyor.com/docs/build-environment/#using-multiple-images-for-the-same-build), which I would think is straightforward.

Reply:
It's obvious once you see things through the eyes of a yaml parser: `- SQLITE_ORM_CXX_STANDARD` shouldn't be on the same level with a dash :roll_eyes: